code/src/Components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "styles/header.css";
+
+export const Header = () => {
+  return (
+      <header className="header">
+        <img src="../img/hero.jpg"
+        className="hero-img"
+        alt="Green leaf in a glass vase with water"
+      />
```

Comment:
Indents are missing, and the first attribute should be on a new row as well, like so:

```
<img
        src="../img/hero.jpg"
        className="hero-img"
        alt="Green leaf in a glass vase with water
/>
```